### ISSUE TITLE
HTML entities in signoff message from hosted Weblate

### ISSUE THREAD

**Issue by ibennetch** [WeblateOrg/weblate#2039]
Hi, over at phpMyAdmin, I just saw a commit from hostted Weblate. In the signoff line, there are HTML entities instead of literal < and > characters:

```
commit c037e682c6ad5591ec02b2ddd04df328fcb8b3d5 (origin/master, origin/HEAD)
Author: Michal Čihař <michal@cihar.com>
Date:   Wed May 23 22:33:14 2018 +0000

    Translated using Weblate (Czech)

    Currently translated at 89.4% (2944 of 3291 strings)

    [CI skip]

    Translation: phpMyAdmin/Development
    Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/cs/
    Signed-off-by: Michal Čihař &lt;michal@cihar.com&gt;
```

**Comment by nijel** [comment]
That was first testing commit, I've fixed that just after it :-) - 90ef2d3b8e7d1091dab749cf0219836e856a775b

**Comment by ibennetch** [comment]
Great!
